demisto_sdk/commands/validate/validators/BC_validators/BC102_is_context_path_changed.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+
+from typing import Iterable, List, Optional, Set
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Integration
+
+
+def is_context_path_changed(integration: Integration) -> dict[str, Set[Optional[str]]]:
+    """
+    This method returns the diff between the integrations versions per command.
+    """
+    final_diff = {}
+    old_integration = integration.old_base_content_object
+    mapping_old_commands = {
+        # Since we're sure old_integration has 'commands' attribute, we ignore it (down casting does not solve it)
+        command.name: command
+        for command in old_integration.commands  # type:ignore[union-attr]
+    }
+    mapping_new_commands = {command.name: command for command in integration.commands}
+    for command in mapping_new_commands:
+        if command in mapping_old_commands:
```

Comment:
```suggestion
    for command in sorted(set(mapping_new_commands).intersection(mapping_old_commands)):
```
can unindent everything onwards